fetch.bs

```diff
@@ -2407,6 +2407,14 @@ with a <i>CORS flag</i> and <i>recursive flag</i>, run these steps:
   have it expose less sensitive information.
 
  <li>
+  <p>If <var>request</var>'s <a for=request>current URL</a>'s <a for=url>scheme</a> is
+  "<code>ftp</code>" and <var>request</var>'s <a for=request>client</a>'s
```

Comment:
Can this "and" be replaced by a comma? (I can do it, just want someone to double check.)